Update scale-iot-solution-azure.md (#4511)
MicrosoftDocs/architecture-center: docs/guide/iot/scale-iot-solution-azure.md

### Azure IoT Hub device provisioning service

> Azure IoT Hub device provisioning service (DPS) is a helper service for IoT Hub that enables zero-touch, just-in-time provisioning to the right IoT hub without requiring human intervention. It has a hard-limit of [10 DPS instances per Azure subscription](/azure/iot-dps/about-iot-dps#quotas-and-limits). 

Reviewer: ```suggestion
Azure IoT Hub device provisioning service (DPS) is a helper service for IoT Hub that enables zero-touch, just-in-time provisioning to the right IoT hub without requiring human intervention. It has a hard-limit of [10 DPS instances per Azure subscription](/azure/iot-dps/about-iot-dps#quotas-and-limits).
```

> It also has a [hard-limit of 1 million registrations](/azure/iot-dps/about-iot-dps#quotas-and-limits) per service instance. You should design with this limit as your design limit to avoid issues in the future.

Reviewer: ```suggestion
The service also has a [hard-limit of 1 million registrations](/azure/iot-dps/about-iot-dps#quotas-and-limits) per service instance. You must address service limits in your workload design limit to avoid issues in the future.
```

> The service also has a [hard-limit of 1 million registrations](/azure/iot-dps/about-iot-dps#quotas-and-limits) per service instance. You must address service limits in your workload design limit to avoid issues in the future.

Reviewer 2: ```suggestion
The service also has a [hard limit of 1 million registrations](/azure/iot-dps/about-iot-dps#quotas-and-limits) per service instance. You must address service limits in your workload design limit to avoid issues in the future.
```

**General DPS provisioning guidance:** You should apply the following recommendations to your DPS deployment, which represent general best practices for this Azure service:

> **Don’t provision on every boot**. The [DPS documentation](/azure/iot-dps/how-to-reprovision#send-a-provisioning-request-from-the-device) specifies that the best practice isn't to provision on every boot. For small use cases, it might seem reasonable to provision at every boot because that’s the shortest path to deployment. However, when scaling up to millions of devices, DPS can become a bottleneck, given [its hard limit of 1,000 registrations per minute per service instance](/azure/iot-dps/about-iot-dps#quotas-and-limits). Even device registration status lookup can be a bottleneck because it has a limit of 5 to 10 polling operations per second. Provisioning results are usually a static mapping to an IoT hub. So, unless your requirements include automated reprovisioning requests, it's best to perform them only on demand. If you anticipate more traffic, scaling out to multiple DPS instances might be the only way to support such scenarios.

Reviewer: ```suggestion
**Don't provision on every boot**. The [DPS documentation](/azure/iot-dps/how-to-reprovision#send-a-provisioning-request-from-the-device) specifies that the best practice isn't to provision on every boot. For small use cases, it might seem reasonable to provision at every boot because that’s the shortest path to deployment. However, when scaling up to millions of devices, DPS can become a bottleneck, given [its hard limit of 1,000 registrations per minute per service instance](/azure/iot-dps/about-iot-dps#quotas-and-limits). Even device registration status lookup can be a bottleneck because it has a limit of 5 to 10 polling operations per second. Provisioning results are usually a static mapping to an IoT hub. So, unless your requirements include automated reprovisioning requests, it's best to perform them only on demand. If you anticipate more traffic, scaling out to multiple DPS instances might be the only way to support such scenarios.
```

**General DPS provisioning guidance:** You should apply the following recommendations to your DPS deployment, which represent general best practices for this Azure service:

> **Don’t provision on every boot**. The [DPS documentation](/azure/iot-dps/how-to-reprovision#send-a-provisioning-request-from-the-device) specifies that the best practice isn't to provision on every boot. For small use cases, it might seem reasonable to provision at every boot because that’s the shortest path to deployment. However, when scaling up to millions of devices, DPS can become a bottleneck, given [its default limit of 1,000 registrations per minute per service instance](/azure/iot-dps/about-iot-dps#quotas-and-limits). Even device registration status lookup can be a bottleneck because it has a limit of 5 to 10 polling operations per second. Provisioning results are usually a static mapping to an IoT hub. So, unless your requirements include automated reprovisioning requests, it's best to perform them only on demand. This limit is a hard-limit and can't be increased, as documented at [
> Quotas and Limits](https://learn.microsoft.com/en-us/azure/iot-dps/about-iot-dps#quotas-and-limits) therefore, scaling out to multiple DPS instances might be the only way to support such scenarios, depending on the anticipated traffic.

Reviewer 3: @Jesusbar  One change you must make before you can merge is replacing the absolute URL with the relative site path. But I also have the following suggestion for simplifying the paragraph as a whole:

```suggestion
**Don’t provision on every boot**. The [DPS documentation](/azure/iot-dps/how-to-reprovision#send-a-provisioning-request-from-the-device) specifies that the best practice isn't to provision on every boot. For small use cases, it might seem reasonable to provision at every boot because that’s the shortest path to deployment. However, when scaling up to millions of devices, DPS can become a bottleneck, given [its hard limit of 1,000 registrations per minute per service instance](/azure/iot-dps/about-iot-dps#quotas-and-limits). Even device registration status lookup can be a bottleneck because it has a limit of 5 to 10 polling operations per second. Provisioning results are usually a static mapping to an IoT hub. So, unless your requirements include automated reprovisioning requests, it's best to perform them only on demand. If you anticipate more traffic, scaling out to multiple DPS instances might be the only way to support such scenarios.
```

Author: Thanks a lot @kgremban !
